src/main/java/com/spotinst/sdkjava/model/ApiElastigroupGetGroupDeploymentStatusResponse.java

```diff
@@ -0,0 +1,109 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+
+public class ApiElastigroupGetGroupDeploymentStatusResponse implements IPartialUpdateEntity {
```

Comment:
Naming convention of this class to be reviewed.

Reply:
Changed to ApiElastigroupGroupDeploymentStatusResponse